### PR TITLE
Fix #216 : Adding additional installation steps for python2

### LINKED ISSUE #216: [BUG]: python2 installation error in ubuntu
### Describe the bug

When I am trying to setup oppia in ubuntu and trying to install python2 I am getting not found error

### URL of the page where the issue is observed.

https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Linux%3B-Python-3%29

### Steps To Reproduce

Follow the installation steps for oppia in wsl2
Then try to install python2

### Expected Behavior

python2 should get install correctly

### Screenshots/Videos

_No response_

### What device are you using?

Desktop

### Operating System

Windows

### What browsers are you seeing the problem on?

Chrome

### Browser version

120.0.6099.129

### Additional context

_No response_

### Tips for developers

Before tackling the bug, please use `git bisect` (see https://git-scm.com/docs/git-bisect) to investigate which PR caused it. If the issue occurred before commit https://github.com/oppia/oppia/commit/c7db0fe, the most effective approach is to utilize the [Python setup](https://github.com/oppia/oppia/wiki/Installing-Oppia/718fd4d0d9ba801142c02e98a499db26ed852c54) for conducting a `git bisect` (You only need to go back as far as commit https://github.com/oppia/oppia/commits/9a334e9). If you need to perform a `git bisect` for any commit beyond that commit, the Docker setup will be suitable. If you find the PR, leave a comment on this issue pointing to it, or just say that it happened before commit 9a334e9 if you could reproduce it there. This will help us fix the issue by reverting the faulty PR.

Also, if this is your first issue, please make sure to follow https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#choosing-a-good-first-issue and https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up before claiming it. Thanks!

### PR DESCRIPTION
**Pull request by kanugurajesh**
Fixes : #216

This PR adds additional installation steps for python2 as the previous ones were not working in some cases. I have included three commands and any one of them is guaranteed to work.

### CODE REVIEW

**Comment by kanugurajesh** [comment]
@seanlip I have added proper comments for the python2 dependency error fix PR.

File: Installing-Oppia-(Linux;-Python-3).md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Alternatively, if you are on Debian/Ubuntu, you can use the `install_prerequisit
 bash scripts/install_prerequisites.sh
 ```
 
-3. Make sure that you have **Python 2** installed, it is needed for the dev server to run. On Ubuntu 20 you can install it using `sudo apt install python2`.
+3. Make sure that you have **Python 2** installed, it is needed for the dev server to run. On Ubuntu 20 you can install it using `sudo apt install python2`. On Ubuntu 18 you can install it using `sudo apt install python-minimal`. If both of them do not work, try using `sudo apt install python2-minimal`.
```
Comment (seanlip) [inline Installing-Oppia-(Linux;-Python-3).md]: If both of them --> If both these commands

**Review by seanlip — CHANGES_REQUESTED** [review]
One suggestion to make it a bit more specific, otherwise looks good -- thanks!

File: Installing-Oppia-(Linux;-Python-3).md (hunk 1 above)
Comment (kanugurajesh) [reply Installing-Oppia-(Linux;-Python-3).md]: okay I will fix it

**Comment by kanugurajesh** [comment]
@seanlip I have made the changes

**Review by seanlip — APPROVED** [review]
Thanks! LGTM.

**Comment by seanlip** [comment]
Also congrats on your first PR to Oppia! :tada:

**Pull request merged**